docs/content/deprecated/Buttons.mdx

```diff
@@ -9,9 +9,6 @@ storybook: '/react/storybook?path=/story/components-button--default-button'
 
 Use [Button](/Button) instead.
 
-For more info on the changes, have a look at the migration guide.
-[Button migration guide](/migration-guide/Button)
```

Comment:
@colebemis Did this point to something earlier? 😅

Reply:
Spent some time looking for the source, but no luck 😢